src/WebJobs.Script.WebHost/Metrics/LinuxContainerMetricsPublisher.cs

```diff
@@ -156,8 +156,6 @@ public void AddFunctionExecutionActivity(string functionName, string invocationI
                 _logger.LogWarning($"Buffer for function activities is full with {_functionActivities.Count} elements. Dropping current batch of function activities");
                 DrainActivities(_currentFunctionActivities, _functionActivities);
             }
-
```

Comment:
Why remove this? Was this causing the problem with logs?

Reply:
No. I just think it's very noisy (every 30 sec for the lifespan of the container) and people are filtering it out from their Kusto queries anyway.